.ratignore

```diff
@@ -23,10 +23,12 @@
 
 # expect script
 .*changepasswd$
+.*winrm.cloud-init$
 
 # doxygen
 .*package-list$
 .*testframeworkdox.txt$
+docs
```

Comment:
Good point.

However, I tested that this seems to work.  I created a file `somedocs` and `docsblah`, both without licenses.  When I ran `rat` it flagged both of those files, which would make it seem like it doesn't treat this change as a regex?

I couldn't seem to find apache-rat documentation for their regex.  If you know where they are could you point me to them?

`$ java -jar ~/tools/apache-rat-0.12/apache-rat-0.12.jar --dir . --exclude-file .ratignore`

```
*****************************************************

 Printing headers for text files without a valid license header...
 
=====================================================
== File: ./docsblah
=====================================================
no license

=====================================================
== File: ./somedocs
=====================================================
no license

=====================================================
```